docs/en/toolkit/toolkit_basics_guides/advanced_config.md

```diff
@@ -178,10 +178,10 @@ To learn about cloning your configuration, as well as other basics of configurat
 
 ## Cloud distribution of pipeline configurations
 
-This guide walks through creating a **centralized configuration**: a single copy of a pipeline configuration that is located in a shared network location and accessed by all users. But if your workforce is distributed across locations, it may not be feasible to share a single copy of a pipeline configuration. To address this scenario, you can use a **distributed configuration**, a pipeline configuration that's uploaded to Shotgun and automatically distributed to users and cached locally. You can [learn more about distributed configuration in our Core API developer documentation](https://developer.shotgunsoftware.com/tk-core/initializing.html#distributed-configurations).
+This guide walks through creating a **centralized configuration**: a single copy of a pipeline configuration that is located in a shared network location and accessed by all users. But, if your workforce is distributed across locations and networks, it may not be feasible to share a single copy of a pipeline configuration. Toolkit allows for this scenario by providing a **distributed configuration** option where a pipeline configuration is uploaded to Shotgun and each time a user accesses a project the configuration is downloaded and cached locally. You can learn more about distributed configurations in our [Core API developer documentation](https://developer.shotgunsoftware.com/tk-core/initializing.html#distributed-configurations).
 
 ## Working with more than one root folder
 
-Ideally your facility would want to be optimized for specific tasks. You can work with more than one root folder to optimize things like video playback for dailies on one server and interactive processing on another. Toolkit allows you to work with more than one storage root, in order to facilitate workflows such as these. You can [learn more about multi-root configurations here]((https://support.shotgunsoftware.com/hc/en-us/articles/219040478-How-do-I-convert-my-project-from-a-single-root-to-a-multi-root-configuration-)). 
+Ideally your facility would want to be optimized for specific tasks. You can work with more than one root folder to optimize things such as video playback for dailies on one server and interactive processing on another. Toolkit allows you to work with more than one storage root in order to facilitate workflows such as these. Check out how to convert from a [single root to a multi-root configuration](https://support.shotgunsoftware.com/hc/en-us/articles/219040478-How-do-I-convert-my-project-from-a-single-root-to-a-multi-root-configuration-) and learn more about a distributed configuration. 
 
```

Comment:
Not sure why you made this change, and there is an error – this has nothing to do with distributed configurations.